.github/ISSUE_TEMPLATE/bug_report_spell.md

```diff
@@ -0,0 +1,96 @@
+labels: 'bug, spell'
+title: '🧙‍♂️ [Bug][Spell] '
+assignees: ''
+---
+<!-- IF YOU DO NOT FILL THIS TEMPLATE OUT, WE WILL CLOSE YOUR ISSUE! -->
+
+<!-- For issues containing a fix, please create a Pull Request following this tutorial: http://www.azerothcore.org/wiki/Contribute#how-to-create-a-pull-request -->
+## 🧙‍♂️ Bug report - Spell 
+
+<!-- WRITE A RELEVANT TITLE -->
+
+
+### Current behaviour
+<!-- Describe the bug in detail. Do not forget to include screenshots. Database to link spells, NPCs, quests etc https://wowgaming.altervista.org/aowow/ -->
+
+
+
+
+### Expected blizzlike behaviour
+<!-- Describe how it should be working without the bug. -->
+
+
+
+
+### Steps to reproduce the problem
+<!-- Describe precisely how to reproduce the bug so we can fix it or confirm its existence:
+ - Which commands to use? Which NPC to teleport to?
+ - Do we need to have debug flags on Cmake?
+ - Do we need to look at the console while the bug happens?
+ - Other steps
+-->
+
+1. 
+2. 
+3. 
+4. ...
+5. Profit
+
+### Extra notes
+<!--
+Any information that can help the developers to identify and fix the issue should be put here.
+Examples:
+- was this bug always present in AzerothCore? if it was introduced after a change, please mention it
+- the code line(s) that cause the issue
+- does this feature work in other server applications (e.g. CMaNGOS, TrinityCore, etc...) ?
+- If this is a crash report, upload the crashlog on https://gist.github.com/
+-->
+
+
+
+
+### Version & Environment
+
+#### AzerothCore commit hash
+<!-- IF YOU DO NOT FILL THIS OUT, WE WILL CLOSE YOUR ISSUE! NEVER WRITE "LATEST", ALWAYS PUT THE ACTUAL VALUE INSTEAD.
+Find the commit hash (unique identifier) by running "git log" on your own clone of AzerothCore or by looking at here https://github.com/azerothcore/azerothcore-wotlk/commits/master -->
+
+Commit Hash:
+
+#### Operating system
+<!-- Windows 7/10, Debian 8/9/10, Ubuntu 16/18 etc... -->
+
+OS Client:
+OS Server:
+
+#### Modules
+<!-- Are you using modules? If yes, list them (note them down in a .txt for opening future issues) -->
+
+
+
+
+#### Other customizations
+<!-- Are you using any extra script?
+ - Did you apply any core patch/diff?
+ - Did you modify your database?
+ - Or do you have other customizations? If yes please specify them here.
+-->
+
+
+
+
+<!-- ------------------------- THE END ------------------------------
+Thank you for your contribution.
+If you use AzerothCore regularly, we really NEED your help to:
+ - TEST our fixes ( http://www.azerothcore.org/wiki/Contribute#how-to-test-a-pull-request )
+ - Report issues
+ - Improve the documentation/wiki
+With your help the project can evolve much quicker!
+-->
+
+
+<!-- NOTE: If you intend to contribute more than once, you should really join us on our discord channel! We set cosmetic ranks for our contributors and may give access to special resources/knowledge to them! The link is on our site http://azerothcore.org/
+-->
```

Comment:
is there a way to avoid this repetition? It can became an issue since we will then have to change the same thing in multiple places. Maybe we can stick with 1 bug report category and let the user specify what's about inside the template.

Apart from this, good job overall. I like the improvements

Reply:
I understand your question that you do not want to have the duplicate content in the different bug report files.

You can not use any sort of scripting inside the issue templates to my knowledge, nor is it possible to to combine template snippets. Therefore the only way would be to blow up the templates to include all possible variations like so:

```
<!--- uncomment the following if you want to report a spell bug --->
<!---
Some spell specific stuff
--->
```

Reply:
unfortunately, I'm not aware of any scripting, so I think the best thing would be to use templates like you just suggested

Reply:
@ulfgebhardt maybe I was not clear, but I meant this:

> Therefore the only way would be to blow up the templates to include all possible variations like so:

to avoid duplicating the same template among different `bug_*` files. So maybe it's better to have only one bug template  file and use the template to include variations (inside it)

Reply:
I disagree

Reply:
@ulfgebhardt I guess we can give it a try, as long as you promise to take care of keeping all the templates in synch :)